mne_neuron/dipole.py

```diff
@@ -162,7 +199,11 @@ def plot(self, ax=None, layer='agg'):
         fig : instance of plt.fig
             The matplotlib figure handle.
         """
-        import matplotlib.pyplot as plt
+        from mne_neuron import sim
+        import matplotlib
+        matplotlib.use("TkAgg")
```

Comment:
I have never used this backend before. Why do you need to enforce this?

Reply:
It could be cased for 'darwin' architectures